layouts/compliance/iso-27001.html

```diff
@@ -0,0 +1,221 @@
+{{ define "main" }}
+
+{{ partial "hero.html" (dict "title" .Title) }}
```

Comment:
Going to leave some comments here for the cloud resource specific pages. I want a slightly different format. 

I think we can remove the 12 PCI DSS Requirements for this type of page to keep it shorter

I want to keep the Key Aspects section. 

After the introduction of the Pulumi services. I think we can have a section that says

"What controls can I put in place to evaluate {service name}"

Then we list out all the items here:  https://docs.aws.amazon.com/securityhub/latest/userguide/ecs-controls.html

Then you can add a CTA with copy above that says "Speak to a Solutions Architect to implement policy as code to manage {service name} for {compliance framework}

Reply:
Oh yeah there should be a section which is "What is {cloud service}" that gives a quick definition

Reply:
I added this What is section above the controls list.